project/SparkBuild.scala

```diff
@@ -765,16 +750,6 @@ object CopyDependencies {
 
 }
 
-object Java8TestSettings {
-  import BuildCommons._
-
-  lazy val settings = Seq(
-    javacJVMVersion := "1.8",
-    // Targeting Java 8 bytecode is only supported in Scala 2.11.4 and higher:
-    scalacJVMVersion := (if (System.getProperty("scala-2.10") == "true") "1.7" else "1.8")
```

Comment:
Thanks @shaneknapp -- so there's no more to the tool chain here than the Scala jars, OK. @vanzin I tried disabling zinc and it still built and ran tests after switching to 2.10. I tried to emulate the job that the master-2.10 Jenkins job would run. I think we're OK on this front?